Shared/Samples/Find route in mobile map package/FindRouteInMobileMapPackageView.Model.swift

```diff
@@ -0,0 +1,195 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension FindRouteInMobileMapPackageView.MobileMapView {
+    /// The view model used to store the geo model and other expensive objects used in the mobile map view.
+    @MainActor
+    class Model: ObservableObject {
+        // MARK: Properties
+        
+        /// The map for the view.
+        let map: Map
+        
+        /// The graphics overlays for the view.
+        var graphicsOverlays: [GraphicsOverlay] {
+            if let routeGraphicsOverlay {
+                return [markerGraphicsOverlay, routeGraphicsOverlay]
+            }
+            
+            return [markerGraphicsOverlay]
```

Comment:
```suggestion
            } else {
                return [markerGraphicsOverlay]
            }
```
We prefer a complete if-else when possible.